src/components/basic/DlButton/DlButton.vue

```diff
@@ -318,10 +333,12 @@ export default defineComponent({
                         '--dl-button-bg': this.colorsObject.ACTIVE.BACKGROUND,
                         '--dl-button-border': this.colorsObject.ACTIVE.BORDER,
                         '--dl-button-text-color-hover':
-                            this.colorsObject.HOVER.TEXT,
+                            this.hoverTextColor || this.colorsObject.HOVER.TEXT,
```

Comment:
i prefer to use the null check operator ??  instead of logical || and since we are changing default to null can you please change this to the nullcheck operator ??  ?

Reply:
done